client/Contexts/ConfigFeatureFlag/Context.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React, { createContext, FunctionComponent, useState } from 'react';
+
+// Temporary config/feature flag definition - to be filled out in later PR with server calls. Add values here and in `strimzi-ui/config` directory
+const config = { example: { value: true } };
+const featureFlags = { capability: { enabled: false } };
+
+// Loading/Error/Trigger Refetch etc
+const contextStates = {
+  loading: false,
+  error: false,
+  isComplete: true,
+  triggerRefetch: () => true,
+};
+
+const ConfigFeatureFlag = createContext({
+  config: {},
+  featureFlags: {},
+  ...contextStates,
+});
+const ConfigFeatureFlagConsumer = ConfigFeatureFlag.Consumer;
+const ConfigFeatureFlagProvider: FunctionComponent = ({
```

Comment:
> I know this will be changed when you add in the data fetching. But I think it would be good to show an example in here of how the updating will work.
> 
> I'm hoping via a mixture of scoping, you will end up calling `triggerRefetch` -> query progress updates the `contextState` -> updates config/featureFlags -> triggering the `useState` to re-render -> Provider to update

The flow will be the query will trigger on mount, and when a result (success or otherwise) is returned, a re render will occur for all children.

The refetch I was expecting would only be used in error scenarios, but there is nothing stopping it being used at any time. The result of the refecth will equally trigger a re render.

As for typing, I belive both here and in the next set of changes the typing is provided/correct. The next PR gets a bit more interesting due to the dynamic shape of what comes back, but the shape will follow what is defined in `config/types`